crates/typst-cli/src/compile.rs

```diff
@@ -97,14 +121,10 @@ pub fn compile_once(
 
 /// Export into the target format.
 fn export(document: &Document, command: &CompileCommand) -> StrResult<()> {
-    match command.output().extension() {
-        Some(ext) if ext.eq_ignore_ascii_case("png") => {
-            export_image(document, command, ImageExportFormat::Png)
-        }
-        Some(ext) if ext.eq_ignore_ascii_case("svg") => {
-            export_image(document, command, ImageExportFormat::Svg)
-        }
-        _ => export_pdf(document, command),
+    match command.output_format()? {
+        OutputFormat::Png => export_image(document, command, ImageExportFormat::Png),
```

Comment:
I wonder whether `ImageExportFormat` enum should be superseded by `OutputFormat`. The match wouldn't be guaranteed to be exhaustive anymore, but does it pull its weight?

Reply:
I don't think so because not all exports are images.